src/main/java/at/bitfire/jtx/JtxContract.kt

```diff
@@ -0,0 +1,1081 @@
+         * Type: [String]
+         */
+        const val PARTSTAT = "partstat"
+
+        /**
+         * Purpose:  To specify whether there is an expectation of a favor of a reply from the calendar user specified by the property value
+         * in this case for the attendee.
+         * see [https://tools.ietf.org/html/rfc5545#section-3.8.4.1] and [https://tools.ietf.org/html/rfc5545#section-3.2.17]
+         * Type: [Boolean]
+         */
+        const val RSVP = "rsvp"
+
+        /**
+         * Purpose:  To specify the calendar users to whom the calendar user specified by the property
+         * has delegated participation in this case for the attendee.
+         * see [https://tools.ietf.org/html/rfc5545#section-3.8.4.1] and [https://tools.ietf.org/html/rfc5545#section-3.2.5]
+         * Type: [String]
+         */
+        const val DELEGATEDTO = "delegatedto"
+
+        /**
+         * Purpose:  To specify the calendar users that have delegated their participation to the calendar user specified by the property
+         * in this case for the attendee.
+         * see [https://tools.ietf.org/html/rfc5545#section-3.8.4.1] and [https://tools.ietf.org/html/rfc5545#section-3.2.4]
+         * Type: [String]
+         */
+        const val DELEGATEDFROM = "delegatedfrom"
+
+        /**
+         * Purpose:  To specify the calendar user that is acting on behalf of the calendar user specified by the property in this case for the attendee.
+         * see [https://tools.ietf.org/html/rfc5545#section-3.8.4.1] and [https://tools.ietf.org/html/rfc5545#section-3.2.18]
+         * Type: [String]
+         */
+        const val SENTBY = "sentby"
+
+        /**
+         * Purpose:  To specify the common name to be associated with the calendar user specified by the property in this case for the attendee.
+         * see [https://tools.ietf.org/html/rfc5545#section-3.8.4.1] and [https://tools.ietf.org/html/rfc5545#section-3.2.18]
+         * Type: [String]
+         */
+        const val CN = "cn"
+
+        /**
+         * Purpose:  To specify reference to a directory entry associated with the calendar user specified by the property in this case for the attendee.
+         * see [https://tools.ietf.org/html/rfc5545#section-3.8.4.1] and [https://tools.ietf.org/html/rfc5545#section-3.2.2]
+         * Type: [String]
+         */
+        const val DIR = "dir"
+
+        /**
+         * Purpose:  To specify the language for text values in a property or property parameter, in this case of the attendee.
+         * see [https://tools.ietf.org/html/rfc5545#section-3.8.4.1] and [https://tools.ietf.org/html/rfc5545#section-3.2.10]
+         * Type: [String]
+         */
+        const val LANGUAGE = "language"
+
+        /**
+         * Purpose:  To specify other properties for the attendee.
+         * see [https://tools.ietf.org/html/rfc5545#section-3.8.4.1]
+         * Type: [String]
```

Comment:
Wie funktioniert das genau? Wenn man einen Sync-Adapter schreibt, was muss man in diese Spalte reinschreiben, um andere Properties zu definieren?

Reply:
Passt, hatte schon Helper-Funktionen geschrieben, die waren im JtxICalObject, die hab ich jetzt in den Contract verschoben, dort sind sie eh richtiger und zentral. Doku hab ich ergänzt.